src/webserver.js

```diff
@@ -48,15 +48,17 @@ module.exports = function({ middlewares }) {
          * Start webserver
          * @param {number} port Port where listen
          * @param {string} host Host where host
+         * @param {string} testDir QUnit tests directory
          * @returns {Promise<void>} Promise about webserver listen
          */
-        listen(port, host) {
+        listen(port, host, testDir) {
             return new Promise((resolve, reject) => {
                 server.listen(port, host, err => {
+                    const testDirectory = testDir ? `${testDir}/` : '';
```

Comment:
can you please normalize the path especially if `testDir` already starts or ends with a slash

Reply:
Thank you. Done.